package.json

```diff
@@ -46,19 +46,19 @@
     "envify": "3.4.0",
     "eslint": "3.19.0",
     "eslint-config-airbnb": "14.1.0",
-    "eslint-config-smaato": "git@github.com:Smaato/eslint-config-smaato.git#v0.1.0",
+    "eslint-config-smaato": "git+https://github.com/smaato/eslint-config-smaato.git#v0.1.0",
     "eslint-plugin-import": "2.2.0",
     "eslint-plugin-jasmine": "1.6.0",
     "eslint-plugin-jsx-a11y": "4.0.0",
     "eslint-plugin-react": "6.9.0",
     "gulp": "3.9.0",
     "gulp-gh-pages": "0.5.4",
     "gulp-replace": "0.5.4",
-    "gulp-tasks": "git@github.com:Smaato/gulp-tasks.git#v0.7.0",
+    "gulp-tasks": "git+https://github.com/smaato/gulp-tasks.git#v0.7.0",
```

Comment:
This particular `gulp-tasks 0.7.0` still points to repository that's [only privately accessible](https://github.com/smaato/gulp-tasks/blob/v0.7.0/package.json#L49) and I do not know what's the resolution is. 

One way could be to do a 0.7.1 release for `gulp-tasks` that points to `eslint-config-smaato` using https instead of ssh and point current `ui-framework` to `gulp-tasks` 0.7.1

Reply:
Please ignore, I tried it inside a new node docker container and was able to install dependencies.